### PR TITLE
Fixes a minor oversight with the seed vault lavaland ruin + minor map tweaks.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
My botany changes are slightly more finalized

## About The Pull Request

With the new changes and added tools to botany, namely the gene shears, I may have slightly forgotten to give them to the seed vault ruin, and as a result, chaos and disorder and panic has swept over the land of the plant people for upwards of 6 weeks. Yeah I uh, probably shouldn't be enabling that.

In addition, I did some small tweaks to item placement within the ruin, and I provided 10 sheets of glass, as they can individually be used to prevent cross-pollination as a luxury. 

## Why It's Good For The Game

image
Parts of this probably shouldn't happen.

## Changelog
:cl:
tweak: The Seed Vault ruin now comes equip with the complete modern arsenal of new botany tools and equipment.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
Can you also make the trays in the seed vault start upgraded at a higher-than-normal tier, to compensate for lifebringers no longer having a better-than-normal gene manipulator?

File: _maps/RandomRuins/LavaRuins/lavaland_surface_seed_vault.dmm [hunk 1]
```diff
@@ -43,9 +46,15 @@
...
-/obj/item/gun/energy/floragun,
-/obj/item/gun/energy/floragun,
-/obj/item/gun/energy/floragun,
+/obj/item/gun/energy/floragun{
+	pixel_z = 6
+	},
+/obj/item/gun/energy/floragun{
+	pixel_y = 4
+	},
+/obj/item/gun/energy/floragun{
```
Comment (ATH1909) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_seed_vault.dmm:55]: I've notcied that you've given lifebringers 3 of each of these kinds of tools... but aren't there 4 lifebringer spawners?

Comment (ArcaneMusic) [reply _maps/RandomRuins/LavaRuins/lavaland_surface_seed_vault.dmm:55]: If you read closer, you'll see that there are 4 total.

File: _maps/RandomRuins/LavaRuins/lavaland_surface_seed_vault.dmm [hunk 2]
```diff
@@ -43,9 +46,15 @@
 /area/ruin/powered/seedvault)
 "aj" = (
 /obj/structure/table/wood,
-/obj/item/gun/energy/floragun,
-/obj/item/gun/energy/floragun,
-/obj/item/gun/energy/floragun,
+/obj/item/gun/energy/floragun{
+	pixel_z = 6
```
Comment (nemvar) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_seed_vault.dmm:50]:
```suggestion
	pixel_y = 6
```

**Comment by HomocidlePancakes** [comment]
Can you move the trash chute to a corner? Will make it easier to reorganize

**Pull request merged**